### PR TITLE
[15.0][MIG] website_cookiebot: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT39017

@chienandalu @pedrobaeza please review!

### CODE REVIEW

File: website_cookiebot/templates/website.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            position="inside"
+        >
+            <div>
+                <span t-if="website.cookiebot_dgid">- <a
+                    href="/legal/cookies-policy"
+                >Cookies Policy</a></span>
+            </div>
+
+        </xpath>
+    </template>
```
Comment (chienandalu) [inline website_cookiebot/templates/website.xml]: Check this other module implementation that will give you a clue on how to make it better (inherit from that other template): https://github.com/OCA/website/blob/14.0/website_legal_page/views/website_legal_main_page.xml#L4-L18

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Review by chienandalu — APPROVED** [review]
:+1:

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**